src/uct/base/uct_md.c

```diff
@@ -494,6 +498,10 @@ void uct_md_set_rcache_params(ucs_rcache_params_t *rcache_params,
     rcache_params->max_regions        = rcache_config->max_regions;
     rcache_params->max_size           = rcache_config->max_size;
     rcache_params->max_unreleased     = rcache_config->max_unreleased;
+    if (rcache_config->purge_on_fork) {
+        rcache_params->flags          = UCS_RCACHE_FLAG_PURGE_ON_FORK;
```

Comment:
1. should be set to 0 if not purge on fork
2. does it mean all registration caches will now have purge-on-fork by default?

Reply:
This will enable purge-on-fork for all md rcaches by default. Though, I should also update gdr_copy too https://github.com/openucx/ucx/blob/7d8e496e9b397bfb7a5a4cd79f2497cf8bdd94f6/src/uct/cuda/gdr_copy/gdr_copy_md.c#L417

Alternatively, I can add the option only to the IB md by defining an extended rcache config type for the IB md.
https://github.com/openucx/ucx/blob/7d8e496e9b397bfb7a5a4cd79f2497cf8bdd94f6/src/uct/ib/base/ib_md.h#L151

Reply:
ok, looks ok as it is now. can u pls squash to one commit?